packages/rocketchat-lib/server/methods/updateMessage.js

```diff
@@ -0,0 +1,36 @@
+Meteor.methods({
+	updateMessage(message) {
+
+		check(message, Match.ObjectIncluding({_id:String}));
+
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'updateMessage' });
+		}
+
+		const originalMessage = RocketChat.models.Messages.findOneById(message._id);
+
+		if (!originalMessage && !originalMessage._id) {
```

Comment:
This line will fail if originalMessage is undefined
Can you change
`if (!originalMessage && !originalMessage._id) {`
by
`if (!originalMessage || !originalMessage._id) {`